### PR TITLE
creating ansible remediation for sssd pam service check

### PR DESCRIPTION
**Pull request by aghassemlouei**
#### Description:

- Created Ansible remediation/fix content tasks based on similar LDAP sssd configurations for DISA STIG RHEL-07-041002

#### Rationale:

- No previous Ansible remediation/fix content existed

- Adds Ansible remediation/fix content similar to #2688

### CODE REVIEW

**Comment by matejak** [comment]
ok to test

File: linux_os/guide/services/sssd/sssd_enable_pam_services/ansible/shared.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# platform = multi_platform_rhel
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = low
+
+- name: "Test for sssd group"
+  shell: grep '\s*\[sssd\/[^]]*]' /etc/sssd/sssd.conf
```
Comment (redhatrises) [inline linux_os/guide/services/sssd/sssd_enable_pam_services/ansible/shared.yml]: Feel like a `regexp` in the `lineinfile` below would be better.

File: linux_os/guide/services/sssd/sssd_enable_pam_services/ansible/shared.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  ignore_errors: yes
+  tags:
+    @ANSIBLE_TAGS@
+  @ANSIBLE_ENSURE_PLATFORM@
+
+- name: "Add sssd group and use pam (if no sssd there)"
+  lineinfile:
+    path: /etc/sssd/sssd.conf
+    create: yes
+    line: "[sssd]\nservices = pam\n"
```
Comment (redhatrises) [inline linux_os/guide/services/sssd/sssd_enable_pam_services/ansible/shared.yml]: What if there are other services in the line?

**Comment by shawndwells** [comment]
bump to @aghassemlouei - Thanks for this work! Any chance you'll be able to edit or comment on feedback from @redhatrises ?

**Comment by dahaic** [comment]
No reply for 6 months. Closing

**Pull request closed without merging**